zellij-server/src/panes/terminal_character.rs

```diff
@@ -619,10 +619,10 @@ impl Display for CharacterStyles {
             if let Some(ansi_code) = self.underline_color {
                 match ansi_code {
                     AnsiCode::RgbCode((r, g, b)) => {
-                        write!(f, "\u{1b}[58;2;{};{};{}m", r, g, b)?;
+                        write!(f, "\u{1b}[58:2::{}:{}:{}m", r, g, b)?;
```

Comment:
While I tested this and it works this way, I wonder - why the two colons? For me it works both ways.

Reply:
According to https://en.wikipedia.org/wiki/ANSI_escape_code#24-bit the format is:

```
ESC[38:2:⟨Color-Space-ID⟩:⟨r⟩:⟨g⟩:⟨b⟩:⟨unused⟩:⟨[CS tolerance](https://en.wikipedia.org/wiki/Color_difference)⟩:⟨Color-Space associated with tolerance: 0 for "CIELUV"; 1 for "CIELAB"⟩ m Select RGB foreground color
ESC[48:2:⟨Color-Space-ID⟩:⟨r⟩:⟨g⟩:⟨b⟩:⟨unused⟩:⟨CS tolerance⟩:⟨Color-Space associated with tolerance: 0 for "CIELUV"; 1 for "CIELAB"⟩ m Select RGB background color
```

where the two colons is the empty `Color-Space-ID` parameter i.e. "unspecified"

Reply:
Gotcha, thanks